### DISCUSSION TITLE
How to Add Vendor Prefixes to support IE11

### DISCUSSION THREAD

**Discussion by gulzaryousaf** [ben-rogerson/twin.macro#236] [Q&A] [answered]
Twin macro doesn't add the vendor prefixes, how can i enable this feature?

**Comment by ben-rogerson** [comment, marked as answer]
Sorry, there's no support for IE11 + Twin. You'll need to use Tailwind CSS v1.9.

**Comment by RasenGUY** [comment]
yeah but support for safari ? and chrome ?
